src/libsync/configfile.cpp

```diff
@@ -381,6 +382,24 @@ quint64 ConfigFile::forceSyncInterval(const QString& connection) const
     return interval;
 }
 
+bool ConfigFile::updateCheckInterval( const QString& connection ) const
+{
+    QString con( connection );
+    if( connection.isEmpty() ) con = defaultConnection();
+    QSettings settings(configFile(), QSettings::IniFormat);
+    settings.beginGroup( con );
+
+    quint64 defaultInterval = 1000*120*60;
```

Comment:
how about 10 hours?  the original bug say once a day.

(It will otherwise create a lot more traffic on the update hour. Like 5 times as much.  Is it not better to spread the updates along the day)